report/src/problem_analysis/understanding.tex

```diff
@@ -1,33 +1,38 @@
 Secondly, we wanted to understand what the stakeholder sees as minimum requirements for a potential solution to their problem. 
-To facilitate this understanding a semi-structured interview was performed afterwards. 
-Due to not knowing the specifics about the problem, we could not create close-ended questions for a structured interview, and thus chose to use a semi structured interview focusing on open-ended questions. This allowed us to ask complementary questions to clarify answers \cite{InterviewsNHS}.
-This interview was based on ideas provided to us by the stakeholder during the initial contact.
+To facilitate this understanding a semi-structured interview was performed afterward. 
+Due to not knowing the specifics of the problem, we could not create close-ended questions for a structured interview and thus chose to use a semi-structured interview focusing on open-ended questions. This allowed us to ask complementary questions to clarify answers \cite{InterviewsNHS}.
+This interview was based on ideas for functional components and UI provided to us by the stakeholder during the initial contact.
 
 
-\section{Understanding the problem} % What constitutes a solution for 
 During the initial interview, the stakeholders provided us with insight into what their current needs are, and why their current system does not satisfy these needs.
-First and foremost, the stakeholder have issues with arranging meetings throughout their day.
-When employees of Mariendal IT schedules meetings, they sometimes experience problems with not having or not being able to locate available rooms.
+
+First and foremost, the stakeholder has issues with arranging meetings throughout their day.
+When employees of Mariendal IT schedule meetings, they sometimes experience problems with not having or not being able to locate available rooms.
 The unavailability is often due to the meeting organizer not registering that a meeting has concluded early.
-This leads to meeting rooms being unattended while still being marked as in-use in their calendar system.
+This leads to meeting rooms being unattended while still being marked as in use in their calendar system.
 Furthermore, the stakeholder will sometimes have unscheduled visits from clients, and will thus need a place to discuss business impromptu. 
 Locating which rooms are available requires the scheduler to use their computer to assess each room's availability.
 Alternatively, the scheduler will visit each meeting room to assess whether it is currently available and then use their computer to book the room if it is available for long enough.
-During the interview they mention that a simple monitor, like a tablet, showing the room's current availability could be a partial solution to this.
+During the interview, they mention that a simple monitor, like a tablet, showing the room's current availability could be a partial solution to this.
```

Comment:
Screen over monitor? To prevent misunderstanding from the double-meaning of monitor?

```suggestion
During the interview, they mention that a simple screen, like a tablet, showing the room's current availability could be a partial solution to this.
```